docs/admin/pool_operations.md

```diff
@@ -104,6 +104,13 @@ The capacity of the pool can be specified in three different ways:
     So in the first example above, specifying `--scm-size=256GB`
     would fail as 256 GB is smaller than the minimum 256 GiB.
 
+!!! warning
+    Concurrent creation of pools using **size percentage** could lead to
+    `ENOSPACE` errors.  Indeed, this operations is not atomic and the overall
```

Comment:
Minor suggested change (up to you & other reviewers on this one):, I propose:

"Indeed, these operations are not atomic and the overall
available size retrieved in a first step could be different from the size
actually available when the second step will be performed (i.e. allocation of space
for the pool)."

Reply:
Agree with this change. Minor nit: "retrieved in a" -> "retrieved in the".

Reply:
Thanks for your review and to correct my frenglish :flushed:

- [ ] Fix frenglish sentence